src/h3/CMakeLists.txt

```diff
@@ -17,3 +17,4 @@ endmacro()
 add_cython_file(util)
 add_cython_file(geo)
 add_cython_file(memview_int)
+add_cython_file(to_multipoly)
```

Comment:
@ankitmehta21 Is there a way to just add all the Cython files automatically?

Reply:
This can be done with `file(GLOB ...)` in CMake but this seems to be recommended against: https://cmake.org/cmake/help/v3.14/command/file.html?highlight=file#filesystem We moved away from using this in uber/h3 for C files.

Reply:
Makes sense. I'll add a comment, just in case the temptation ever arises again.